Automattic/jetpack
# Contact Info Block: Fix styling issues for FSE compatibility

Related to: 256-gh-Automattic/view-design

This change updates the CSS for the editor view of the Contact Info block to create consistency between the post editor and the new FSE site editor. The existing editor CSS depends on `forms.css` being present to provide the border-radius, and the gap between each form field occurs by default because the elements were set to `inline-block`. This change makes the gap explicit using a margin, and sets the input fields to be displayed as blocks. It also explicitly prevents the form field from being manually resized.

#### Screenshots

Using the site editor with Twenty Twenty One Blocks theme:

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/113670220-2ca50480-96f8-11eb-8a85-74be1f5b5729.png) | ![image](https://user-images.githubusercontent.com/14988353/113670243-329ae580-96f8-11eb-9cf1-92042bf69003.png) |

#### Reference screenshot from the post editor

This is how the block looks in the normal post editor (this is the view we're trying to match):

![image](https://user-images.githubusercontent.com/14988353/113670318-4d6d5a00-96f8-11eb-866d-dd38f77096b0.png)

#### Changes proposed in this Pull Request:

* Update Contact Info editor styling to deliberately set the border radius for the input fields
* Force the resize controls for the textarea element to not be displayed
* Set the vertical margin between input fields

#### Jetpack product discussion


256-gh-Automattic/view-design

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply this change using an FSE-compatible theme and from the site editor (e.g. wp-admin/admin.php?page=gutenberg-edit-site) test the Contact Info block
* From the standard post editor, compare and ensure the editor UI looks basically the same
* Try with a couple of other themes and non-FSE themes to ensure this doesn't introduce regressions / degrade the editing experience
